src/main/java/edu/harvard/iq/dataverse/DataFileServiceBean.java

```diff
@@ -1435,4 +1445,15 @@ public boolean isReplacementFile(DataFile df) {
         }
         return false;
     }  // end: isReplacementFile
+    
+    public List<Long> selectFilesWithMissingOriginalTypes() {
+        Query query = em.createNativeQuery("SELECT f.id FROM datafile f, datatable t where t.datafile_id = f.id AND t.originalfileformat='" + MIME_TYPE_TAB + "' ORDER BY f.id");
```

Comment:
No, the above isn't going to work as a TypedQuery; OK, I did build and ran it, it actually fails with multiple errors - "cannot be resolved to a valid type" and "abstract schema type ... is unknown", etc. 
But, my main point still is, even if it worked as a typedquery, it's not clear to me that it would be adding any real value... seeing how it is a one-time thing... I really want to move this thing along, so, since you said you were ok with it going into QA, I'm taking you up on it, and doing just that.